Assets/Game/Addons/UnityConsole/Console/Scripts/DefaultCommands.cs

```diff
@@ -26,8 +26,7 @@ public class DefaultCommands : MonoBehaviour
     {
         public static bool showDebugStrings = false;
 
-        void Start()
-        {
+        void Start() {
```

Comment:
This repository uses the original style.

The entire file was changed and then reverted without apparent reason. Can you try to rebase and cleanup commit history a bit?